composer.json

```diff
@@ -21,9 +21,9 @@
     "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
     "oat-sa/extension-tao-ltideliveryprovider" : ">=12.0.0",
     "oat-sa/generis" : ">=14.0.0",
-    "oat-sa/tao-core" : ">=47.0.0",
+    "oat-sa/tao-core" : ">=48.39.0",
     "oat-sa/extension-tao-delivery" : ">=15.0.0",
-    "oat-sa/extension-tao-lti" : ">=13.2.0",
+    "oat-sa/extension-tao-lti" : ">=15.4.0",
```

Comment:
Unfortunately, [it's still impossible](https://devkit-oat-demo.dev.gcp-eu.taocloud.org/platform/message/launch/submission-review?registration=infosignTestToolRegistration&user_list=c3po&ags_scopes%5B0%5D=https://purl.imsglobal.org/spec/lti-ags/scope/score&ags_line_item_url=https://devkit-oat-demo.dev.gcp-eu.taocloud.org/platform/service/ags/context_id/lineitems/item_id&for_user_type=other&for_user_id=user_id_1&submission_review_url=https://delivery.test-infosign.playground.kitchen.it.taocloud.org/ltiTestReview/ReviewTool/launch1p3?delivery%3Dhttp%253A%252F%252Fdelivery.test-infosign.playground.kitchen.it.taocloud.org%252Ftao.rdf%2523i617bd2cea4545185e034c58924ecefcb&claims=%7B%0D%0A%20%20%20%20%22https://purl.imsglobal.org/spec/lti/claim/roles%22:%20%5B%0D%0A%20%20%20%20%20%20%20%20%22http://purl.imsglobal.org/vocab/lis/v2/membership%23Mentor%22%0D%0A%20%20%20%20%5D,%0D%0A%20%20%20%20%22https://purl.imsglobal.org/spec/lti/claim/custom%22:%20%7B%0D%0A%20%20%20%20%20%20%20%20%22custom_show_score%22:%20%22true%22,%0D%0A%20%20%20%20%20%20%20%20%22custom_show_correct%22:%20%22true%22%0D%0A%20%20%20%20%7D%0D%0A%7D) to utilize the Mentor and Instructor roles.

It seems we have to extend the ontology model where the roles are stored by TAO.
See [here](https://github.com/oat-sa/extension-tao-lti/pull/304/files#diff-f4298c9e7aea19430744b6ecf2fcb699cb3e6bb87affbae305195966fdd02e30) and [here](https://github.com/oat-sa/extension-tao-lti/pull/304/files#diff-6a360aa65a37ef0c0f0417a1d93e106e09904247c2673b48102617c5e5cd4d1b) for reference.